packages/suite/src/views/suite/bridge/index.tsx

```diff
@@ -133,6 +133,7 @@ const InstallBridge = (props: Props) => {
     };
 
     const target = selectedTarget || data.target;
+    const isLoading = !props.transport || process.env.SUITE_TYPE === 'desktop';
```

Comment:
maybe this never happens as there is check for transport in SuiteWrapper/index line 32?